### PR TITLE
[MM-42369] Add status update

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR adds status update component to the `/editor`

https://user-images.githubusercontent.com/2340127/167252206-e3df4929-3c90-4e96-b1e9-c57aa826eab5.mp4

#### Ticket Link
  Fixes: https://mattermost.atlassian.net/browse/MM-42369

#### Checklist

- ~~[ ] Telemetry updated~~
- ~~[ ] Gated by experimental feature flag~~
- ~~[ ] Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/backstage/playbook_editor/outline/webhooks_input.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+import {useClientRect} from 'src/hooks';
+import Dropdown from 'src/components/dropdown';
+import {CancelSaveButtons} from 'src/components/checklist_item/inputs';
+
+import {Placeholder} from './update_timer_selector';
+import {moveRect} from './broadcast_channels_selector';
+
+type Props = {
+    webhookOnStatusUpdateURLs: string[];
```
Comment (crspeller) [inline webapp/src/components/backstage/playbook_editor/outline/webhooks_input.tsx:16]: Probably no need to be this specific. URLs is good enough.

File: webapp/src/components/backstage/playbook_editor/outline/update_timer_selector.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            if (matched) {
+                value = matched;
+            } else {
+                options.push(value);
+            }
+            options.sort((a, b) => ms(a.value) - ms(b.value));
+        }
+
+        return {options, value};
+    }, [props]);
```
Comment (crspeller) [inline webapp/src/components/backstage/playbook_editor/outline/update_timer_selector.tsx:45]: Does this work? I think we should specify the dependencies explicitly regardless. What if someone adds a new prop?

File: webapp/src/components/backstage/playbook_editor/outline/broadcast_channels_selector.tsx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+            filteredChannel = channel;
+        } else {
+            filteredChannel = {display_name: '', id} as Channel;
+        }
+        result.push(filteredChannel);
+    });
+    return result;
+};
+
+const sortChannels = (allChannels: Channel[], selectedChannelIds: string[]): Channel[] => {
```
Comment (crspeller) [inline webapp/src/components/backstage/playbook_editor/outline/broadcast_channels_selector.tsx:52]: Looks more like this is splitting the channels not sorting them. Maybe `splitChannels`?

**Review by crspeller — COMMENTED** [review]
Just a couple nits.

**Review by calebroseland — APPROVED** [review]

**Pull request merged**